src/components/Sweep/SweepExecute.tsx

```diff
@@ -0,0 +1,222 @@
+import { useCallback, useState, useEffect } from 'react';
+import { Form, Input, Button, notification, Typography, Space } from 'antd';
+import { ethers } from 'ethers';
+import styled from 'styled-components';
+import debounce from 'lodash/debounce';
+
+import { ethereumAddressValidator, shortAddress } from '../../utils';
+import { useMetamask } from '../../hooks/useMetamask';
+import { ChainId } from '../../constants';
+import { contractBalanceOf, contractTransferFrom, contractAllowance } from './magicContractHelper';
+import { useSweepStore } from './useSweep';
+
+const { Text } = Typography;
+
+function SweepExecute() {
+  const [loading, setLoading] = useState(false);
+
+  const [amount, setAmount] = useState('');
+
+  const storedFromAddr = localStorage.getItem('fromAddr')?.toString();
+  const storedToAddr = localStorage.getItem('toAddr')?.toString();
+
+  const { fromAddr, setFromAddr } = useSweepStore();
+  const { toAddr, setToAddr } = useSweepStore();
```

Comment:
just get all props you need in one `useSweepStore` call, no need to call it multiple times